src/endpointParsers.js

```diff
@@ -42,12 +42,14 @@ export function processLiveAvailability(response) {
     if (typeof data.default === 'undefined') return 'outage';
     return data.live_lookups.success ? 'up' : 'outage';
   });
-};
+}
 export function processLibraryDrupal(response) {
-  return response.body.getReader()
-    .read()
-    .then((data) => {
-      console.log(data);
-      return data.components[1].status === 'operational' ? 'up' : 'outage';
-    });
+  return response.text().then(function(data) {
+    let lastLine = data.split("\n").pop();
+    if (lastLine.match(/^all health checks successfull/i)) {
```

Comment:
Not sure if (there was an | this is the) agreed upon way to determine if the library website is up or not.

Reply:
Neither am I, same with the other health checks though, tbh.